### PR TITLE
[tiptap-plugins]internal: inject services from plugin entity

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

File: packages/ricos-types/web/src/editorPluginTypes.ts [hunk 1]
```diff
@@ -275,3 +264,12 @@ export interface IPluginToolbarButton {
 
   unregister: () => void;
 }
+
+export type PluginServices = {
```
Comment (lxgreen) [inline packages/ricos-types/web/src/editorPluginTypes.ts:268]: we already [have](https://github.com/wix/ricos/blob/master/packages/ricos-types/web/src/index.ts#L40) it, please reuse

**Review by lxgreen — CHANGES_REQUESTED** [review]
niiiice! just see the type

File: packages/ricos-types/web/src/editorPluginTypes.ts (hunk 1 above)
Comment (danarish4) [reply packages/ricos-types/web/src/editorPluginTypes.ts:268]: I thought of reusing it as `Partial<PluginServices>`
But PluginServices type is missing `modalService` & `t`

Do you think the type should be 
`Partial<PluginServices> & { t: TranslationFunction, modalService: ModalService } `

Comment (lxgreen) [reply packages/ricos-types/web/src/editorPluginTypes.ts:268]: nope, just let's add **all** available services to RicosServices, and use it everywhere

**Pull request merged**